addon/components/active-link.js

```diff
@@ -24,6 +20,19 @@ export default Ember.Component.extend({
 
       this.set('childLinkViews', Ember.A(childLinkViews));
     });
-  }
+  },
+
+  hasActiveLinks: Ember.computed('childLinkViews.@each.active', function(){
+    return this.get('childLinkViews').isAny('active');
+  }),
+
+  activeClass: Ember.computed('childLinkViews.@each.active', function(){
+    let activeLink = this.get('childLinkViews').findBy('active');
+    return (activeLink ? activeLink.get('active') : 'active');
```

Comment:
This is great, it's good that it takes it from the child by default or you can override it